File: app/src/main/java/ua/com/foxminded/locationtrackera/data/UserLocation.java
package ua.com.foxminded.locationtrackera.data;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class UserLocation {

    @PrimaryKey(autoGenerate = true)
    public int locationId;
    public double latitude;
    public double longitude;
    public double date;

    public UserLocation(double latitude, double longitude, double date) {
        this.latitude = latitude;
        this.longitude = longitude;
        this.date = date;
    }
}

File: models/src/main/java/ua/com/foxminded/locationtrackera/models/locations/UserLocation.kt
package ua.com.foxminded.locationtrackera.models.locations

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class UserLocation(
    var latitude: Double = 0.0,
    var longitude: Double = 0.0,
    var date: Long = 0
) {
    @PrimaryKey(autoGenerate = true)
    var locationId: Int = 0
}
